using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -379,24 +381,24 @@ AEPCore.collectLaunchInfo(userInfo)
 
 For more information, see the following:
 
-* [Troubleshooting guide](https://marketing.adobe.com/resources/help/en_US/mobile/android/c_troubleshooting-push-messaging.html) \(Android\)
-* [Troubleshooting guide](https://marketing.adobe.com/resources/help/en_US/mobile/ios/c_troubleshooting-push-messaging.html) \(iOS\)
+* [Android troubleshooting guide](https://experienceleague.adobe.com/docs/mobile-services/android/messaging-android/push-messaging/c-troubleshooting-push-messaging.html)
+* [iOS troubleshooting guide](https://experienceleague.adobe.com/docs/mobile-services/ios/messaging-ios/push-messaging/c-troubleshooting-push-messaging.html)
 
 ### Set up in-app messaging
 
-This feature allows you to deliver in-app messages that are triggered from any analytics data or event. After the implementation, messages are dynamically delivered to the app and do not require a code update. In-app messages are created in Mobile Services. For more information, see [Create an in-app message](https://marketing.adobe.com/resources/help/en_US/mobile/?f=t_in_app_message).
+This feature allows you to deliver in-app messages that are triggered from any analytics data or event. After the implementation, messages are dynamically delivered to the app and do not require a code update. In-app messages are created in Mobile Services. For more information, see the [create an in-app message](https://experienceleague.adobe.com/docs/mobile-services/android/messaging-android/inapp-messaging/messaging.html) tutorial.
 
 To set up your app for in-app messages, implement the following instructions. You can complete these steps even if you have not yet defined any messages in Mobile Services. After you define messages, they are delivered dynamically to your app and displayed without an app store update.
 
 {% tabs %}
 {% tab title="Android" %}
-1. Update the `AndroidManifest.xml` file to declare the full screen activity and enable the Message Notification Handler.
+Update the `AndroidManifest.xml` file to declare the full screen activity and enable the Message Notification Handler.
 
 #### Java
 
-If you are using Fullscreen message or local notification, update the `AndroidManifest.xml` with:
+If you are using a fullscreen message or a local notification, update the `AndroidManifest.xml` with the following:
```

Comment:
If you are using `fullscreen message` functionality or `local notification` functionality, update ...